docs/introduction/faq-and-common-issues.md

```diff
@@ -20,7 +20,7 @@ For more detailed information about the Shopsys Framework, please see [Shopsys F
 - [Why are you using entity data instead of entities for Symfony forms?](#why-are-you-using-entity-data-instead-of-entities-for-symfony-forms)
 - [What is the configuration file `services_test.yml` good for?](#what-is-the-configuration-file-services_testyml-good-for)
 - [How to change the behavior of the product search on the front-end?](#how-to-change-the-behavior-of-the-product-search-on-the-front-end)
-- [Why are e-mails not sent immediately but at the end of the script](#why-are-e-mails-sent-before-end-of-the-script-and-not-immediately)
+- [Why are emails not sent immediately but at the end of the script](#why-are-emails-sent-before-end-of-the-script-and-not-immediately)
```

Comment:
I notice one more usage of "e-mail" in`docs/kubernetes/how-to-set-smtp-server-container.md`

Reply:
fixed